src/lib/utils/SynapseContext.tsx

```diff
@@ -11,8 +11,10 @@ import { ThemeOptions } from '@mui/material'
 export const defaultQueryClientConfig: QueryClientConfig = {
   defaultOptions: {
     queries: {
-      staleTime: 60 * 1000, // 60s
+      staleTime: 1000 * 60 * 5, // 5 min
+      cacheTime: 1000 * 60 * 30, // 30 min
```

Comment:
So data will be considered "fresh" (won't be auto-refetched) for 5 minutes, and stale data could appear before being immediately refetched for 30 minutes.